build.sbt

```diff
@@ -693,6 +693,7 @@ lazy val docs = project
     sharedSettings,
     skip.in(publish) := true,
     moduleName := "metals-docs",
+    munitBucketName := Some("scala-metals-test-reports"),
```

Comment:
You may want to leave a comment on what Google account owns this bucket to help the next person investigate the failing CI 😅

Reply:
I think I should be able to give more people access to it. I will try to do that later if it works, but I added a comment for now.